### PR TITLE
Fix broken URLs

### PR DESCRIPTION
**Pull request by wilzbach**
Sister PR to https://github.com/dlang/phobos/pull/4413

### CODE REVIEW

File: spec/version.dd [hunk 1]
```diff
@@ -295,27 +295,27 @@ $(H3 $(LEGACY_LNAME2 PredefinedVersions, predefined-versions, Predefined Version
 	$(TROW $(ARGS $(D BigEndian)) , $(ARGS Byte order, most significant first))
 	$(TROW $(ARGS $(D ELFv1)) , $(ARGS The Executable and Linkable Format v1))
 	$(TROW $(ARGS $(D ELFv2)) , $(ARGS The Executable and Linkable Format v2))
-	$(TROW $(ARGS $(D D_Coverage)) , $(ARGS $(DPLLINK code_coverage.html, Code coverage analysis) instrumentation
+	$(TROW $(ARGS $(D D_Coverage)) , $(ARGS $(DDLLINK code_coverage.html, Code coverage analysis) instrumentation
```
Comment (aG0aep6G) [inline spec/version.dd]: Typo here: `DDLLINK` -> `DDLINK`.

**Comment by wilzbach** [comment]
> Typo here: DDLLINK -> DDLINK.

Thanks @aG0aep6G - fixed. I did a full rebase because the diff is small and I believe it won't undergo such an epic review battle as the last PRs ;-)

File: gsoc2012.dd [hunk 2]
```diff
@@ -30,8 +30,7 @@ $(P Current mentors: Walter Bright (Digital Mars), Andrei Alexandrescu
 (Facebook), David Simcha (Johns Hopkins University), Alex R&oslash;nne
 Petersen (MCI), Adam Wilson (Prospective Software), Jens Mueller
 (Friedrich Schiller University, Jena) Jonas Drewsen (Unity
-Technologies), and $(LINK2 http://www.russel.org.uk/, Russel Winder)
-(Independent Consultant).)
+Technologies), and Russel Winder (Independent Consultant).)
```
Comment (aG0aep6G) [inline gsoc2012.dd]: Google still lists russel.org.uk. Maybe it's just a temporary blackout. @russel?

File: spec/version.dd [hunk 3]
```diff
@@ -295,27 +295,27 @@ $(H3 $(LEGACY_LNAME2 PredefinedVersions, predefined-versions, Predefined Version
 	$(TROW $(ARGS $(D BigEndian)) , $(ARGS Byte order, most significant first))
 	$(TROW $(ARGS $(D ELFv1)) , $(ARGS The Executable and Linkable Format v1))
 	$(TROW $(ARGS $(D ELFv2)) , $(ARGS The Executable and Linkable Format v2))
-	$(TROW $(ARGS $(D D_Coverage)) , $(ARGS $(DPLLINK code_coverage.html, Code coverage analysis) instrumentation
+	$(TROW $(ARGS $(D D_Coverage)) , $(ARGS $(DDLINK code_coverage.html, Code coverage analysis, Code coverage analysis) instrumentation
```
Comment (aG0aep6G) [inline spec/version.dd]: No ".html" suffix with `DDLINK`.

**Comment by wilzbach** [comment]
> Google still lists russel.org.uk. Maybe it's just a temporary blackout. @russel?

Seems like it was. Added the link back in again.

So is this ready to be squashed?

**Pull request merged**